stingray/varenergyspectrum.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import division
+import numpy as np
+from stingray.gti import check_separate, cross_two_gtis
+from stingray.lightcurve import Lightcurve
+from stingray.utils import assign_value_if_none, simon
+from stingray.crossspectrum import AveragedCrossspectrum
+import six
+
+
+class VarEnergySpectrum(object):
+    def __init__(self, events, freq_interval, energy_spec, ref_band,
+                 bin_time=1, use_pi=False, log_distr=False,
+                 segment_size=None, events2=None):
+
+        """Generic variability-energy spectrum.
```

Comment:
Is this class ever called or should it just be a super-class defining some functionality? 
If it's the latter, one may consider the use of the `ABCMeta` super-class, and also perhaps a note somewhere in the documentation not to call this class.